cmd/utils/flags.go

```diff
@@ -742,6 +742,11 @@ var (
 		Usage: "Set the value transfer recovery interval (seconds)",
 		Value: 60,
 	}
+	DefaultTxGasLimitFlag = cli.Uint64Flag{
+		Name:  "defaultgaslimit",
+		Usage: "Set the default value of gas limit for transactions made by bridges",
+		Value: 5000000,
```

Comment:
Just out of curiosity, where does this value come from?

Reply:
same comment, marked as resolved.